ui/src/album/AlbumGridView.js

```diff
@@ -97,29 +100,64 @@ const getColsForWidth = (width) => {
 }
 
 const Cover = withContentRect('bounds')(
-  ({ album, measureRef, contentRect }) => {
+  ({ album, isLoaded, measureRef, contentRect }) => {
     // Force height to be the same as the width determined by the GridList
     // noinspection JSSuspiciousNameCombination
     const classes = useCoverStyles({ height: contentRect.bounds.width })
     return (
       <div ref={measureRef}>
-        <img
-          src={subsonic.getCoverArtUrl(album, 300)}
-          alt={album.name}
-          className={classes.cover}
-        />
+        {isLoaded ? (
+          <img
+            src={subsonic.getCoverArtUrl(album, 300)}
+            alt={album.name}
+            className={classes.cover}
+          />
+        ) : (
+          <div
+            className={classes.cover}
+            style={{ backgroundColor: '#222', borderRadius: 4 }}
```

Comment:
We should move this (and all other styles) to the `useStyles` variable.Also, we should avoid using hardcoded colours ex: `#222`), to be theme-agnostic and avoid this:
<img width="1542" alt="Screen Shot 2021-09-20 at 9 20 24 PM" src="https://user-images.githubusercontent.com/331353/134098172-a4d17fef-18de-4bdb-8be3-f6ac023123cd.png">